src/track/trackmetadatataglib.cpp

```diff
@@ -967,6 +1032,20 @@ void readTrackMetadataFromID3v2Tag(TrackMetadata* pTrackMetadata,
         pTrackMetadata->setAlbumArtist(toQStringFirstNotEmpty(albumArtistFrame));
     }
 
+	QString rating;
+    const TagLib::ID3v2::FrameList popmRatingFrameList(tag.frameListMap()["POPM"]);
+	if (!popmRatingFrameList.isEmpty()) {
+		rating = QString::number(dynamic_cast<TagLib::ID3v2::PopularimeterFrame*>(popmRatingFrameList.front())->rating());
```

Comment:
The point for the duplicaton was to take the first rating found and then check all of them if there is a mixxx specific frame. Since this is only truly a problem for MP3, an alternative would be to forget having the mixxx specific email and just reading writing the 1st frame. 

That is really a decision that should be made by people more familiar with the trajectory of the project.